SearchPathExtender.h

```diff
@@ -35,6 +35,7 @@
 // Standard includes
 #include <algorithm>
 #include <cstdlib>
+//#include <iostream>
```

Comment:
@rpavlik Should I also remove the commented-out `std::cout`s, too? Or did you mean leave the `#include` uncommented?